index.js

```diff
@@ -1039,6 +1039,62 @@
       [Fn(a, $.Boolean), m(a), m(a)],
       Z.filterM);
 
+  function Array$findIndex(pred, xs) {
+    for (var idx = 0; idx < xs.length && pred(xs[idx]); idx += 1);
+    return idx;
+  }
```

Comment:
I would expect `Array$findIndex(x => x === 42, [1, 2, 3])` to evaluate to `-1` rather than to `0`. It seems this function isn't quite `Array#findIndex` after all.